internal/pkg/install/install.go

```diff
@@ -43,7 +51,7 @@ func Install(args string, data *userdata.UserData) (err error) {
 	var previousMountPoint string
 	for dest, urls := range dataURLs {
 		if dest != previousMountPoint {
-			log.Printf("Downloading assets for %s\n", dest)
```

Comment:
Most of the rest of the codebase uses lowercase. Figured I'd align this with that convention and we can discuss a style we agree on and refactor later.